client/src/components/helpers/link.tsx

```diff
@@ -9,6 +9,7 @@ interface LinkProps {
   state?: Record<string, unknown>;
   to: string;
   onClick?: () => void;
+  id?: string;
```

Comment:
Where is this used?

Reply:
I should have added the id in the HTML. As in the article, Arabic links can have dots in top of letter, so I am going to add ID for links to include specific CSS. 

Used id instead of className, because Id has higher specificity, so extensions should ignore it

Reply:
This would be over-customizing.

In general, we would not need to pass additional any classes, or ids to improve the RTL compatibility of our layout.

Other that adding ids, there are other ways to to change CSS's order of precedence.  

For instance, `[dir=rtl] a` would take precedence over `a`

Reply:
```suggestion
```

Reply:
I have added the link back, because I am juggling between multiple tasks, I was confused. 

In here the link id is used, but it isn't used else where, so I made it optional.
https://github.com/freeCodeCamp/freeCodeCamp/pull/48144/files#:~:text=%27/learn%27%3E-,%3CLink%20id%3D%27universal%2Dnav%2Dlogo%27%20to%3D%27/learn%27%3E,-Check%20failure%20on

so this one isn't related to arabic curriculum, for now

```suggestion
  id?: string;

Reply:
I can't apply suggestion to the file anymore 🤣

Reply:
I just unresolved this since I think we're still discussing a little.